setup.py

```diff
@@ -54,13 +54,19 @@ def no_cythonize(extensions, **_ignore):
 sources = [x for x in sources if not x.endswith(('htsfile.c', 'tabix.c', 'bgzip.c'))]
 sources.append('cyvcf2/helpers.c')
 
+extra_objects = []
+LIBDEFLATE = os.getenv("LIBDEFLATE")
+if LIBDEFLATE is not None:
+    extra_objects.append(LIBDEFLATE)
+
 extensions = [Extension("cyvcf2.cyvcf2",
                         ["cyvcf2/cyvcf2.pyx"] + sources,
                         libraries=['z', 'bz2', 'lzma', 'curl', 'ssl'] + (['crypt'] if platform.system() != 'Darwin' else []),
```

Comment:
why not just add `deflate` to libraries if LIBDEFLATE is not None?

Reply:
That does dynamic linking on my system. Does the ciwheelbuild pull that in dynamically or statically? Maybe it just works there?